signac/core/collection_buffered_json.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Implements Buffered-JSON-backend."""
+import os
+import json
+import errno
+import uuid
+import logging
+import hashlib
+
+from .synced_collection import SyncedCollection
+from .buffered_collection import BufferedSyncedCollection
+from .collection_json import JSONCollection
+from .syncedattrdict import SyncedAttrDict
+from .synced_list import SyncedList
+from .caching import get_cache
+from .buffered_collection import get_buffer_force_mode
+
+logger = logging.getLogger(__name__)
+
+_JSON_CACHE = None
+_JSON_META = dict()
+_JSON_HASHES = dict()
+
+
+def get_json_cache():
+    """Return reference to JSON-Cache """
+    global _JSON_CACHE
+    if _JSON_CACHE is None:
+        _JSON_CACHE = get_cache()
+    return _JSON_CACHE
```

Comment:
Is there a reason not to just initially set `_JSON_CACHE = get_cache()`, when the variable is first defined, then access it directly everywhere inside `BufferedJSONCollection`? That seems cleaner and easier than having this function and an extra class variable `_cache` that just always references the global cache anyway.

Reply:
This is leftover from my old implementation. I'll change that.